deepchem/feat/base_classes.py

```diff
@@ -421,3 +421,23 @@ class UserDefinedFeaturizer(Featurizer):
   def __init__(self, feature_fields):
     """Creates user-defined-featurizer."""
     self.feature_fields = feature_fields
+
+
+class DummyFeaturizer(Featurizer):
```

Comment:
Can you add this to `docs/source/api_reference/featurizers.rst`? That will ensure the docs render the featurizer